src/system/RoutesModule/Form/Type/Base/AbstractRouteType.php

```diff
@@ -173,10 +167,9 @@ public function addEntityFields(FormBuilderInterface $builder, array $options)
                 'class' => '',
                 'title' => $this->__('Enter the host of the route')
             ],
-            'required' => false
-            ,
+            'required' => false,
         ]);
-
+        
         $listEntries = $this->listHelper->getEntries('route', 'schemes');
```

Comment:
any way to eliminate these useless spaces/tabs from the generator? overtime my IDE opens these, it corrects them and then the generator switches them back.